### PR TITLE
Update invoices section, cleanup

### PR DESCRIPTION
**Pull request by rocioar**
This updates the invoicing section. And joins the text together of the time management and invoices section.

I've imported the course into the dev course. It easier to visualize it here: https://studio-courses.opencraft.hosting/container/block-v1:OpenCraft+onboarding-dev+course+type@vertical+block@time_management_and_invoices_invoices

https://github.com/open-craft/courses/archive/rocio/invoices-update-cleanup.tar.gz

### CODE REVIEW

**Comment by clemente** [comment]
@rocioar For the next time, please use two separate commits (one to change the structure, another to update the text), so that it's easy to review what changed.

:+1: but could you use the same capitalization style for *Time management* as in the rest of the units? (now: *Time Management*). After this, you can squash and merge.

Btw I reimported it [to the dev course](https://studio-courses.opencraft.hosting/container/block-v1:OpenCraft+onboarding-dev+course+type@vertical+block@time_management_and_invoices_invoices) in order to see it.
I checked the new text about invoices (now automated), looks fine and much simpler than before.

File: sequential/time_management_and_invoices.xml [hunk 1]
```diff
@@ -1,7 +1,310 @@
...
+      </blockquote>
+      <p>
+        Q: How often (and when) should we invoice OpenCraft, and when can I expect payment?
+        What details should be included in the invoice?
+      </p>
+      <p>
+      </p>
+      <blockquote>
+        <p>
+          You'll learn about it soon in details in another unit of this section.
```
Comment (antoviaque) [inline sequential/time_management_and_invoices.xml]: Link to that section?

File: sequential/time_management_and_invoices.xml [hunk 2]
```diff
@@ -1,7 +1,310 @@
...
+          easier (don't have to remember to check the clock every time when I start:)).
+        </p>
+        <p>
+          In order to keep things manageable, I log time in 0.5 hour chunks.
+          I don't keep track of each individual 5 minute email -
+          if I'm answering emails for 30 minutes, I will log all 30 minutes
+          to the client for which I spent most of that 30 minutes.
+          In theory the logged time should even up over time.
+        </p>
+      </blockquote>
```
Comment (antoviaque) [inline sequential/time_management_and_invoices.xml]: Also include a link to https://forum.opencraft.com/t/working-remotely-tips-and-tricks/112/ ?

**Comment by rocioar** [comment]
@clemente @antoviaque I've updated the course based on your feedback, let me know if sounds good to you!

**Comment by clemente** [comment]
@rocioar :+1:, and because of the timebox we don't have time to discuss it more, so if there aro more improvements they can be done later.

**Comment by antoviaque** [comment]
@rocioar @clemente Thanks for the changes! :+1:

**Pull request merged**